src/NuGet.Core/NuGet.Frameworks/FrameworkNameProvider.cs

```diff
@@ -73,6 +74,8 @@ public FrameworkNameProvider(IEnumerable<IFrameworkMappings> mappings, IEnumerab
             _profilesToShortName = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
             _identifierShortToLong = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
             _profileShortToLong = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
+            _platformToShortName = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
+            _platformShortToLong = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
             _portableFrameworks = new Dictionary<int, HashSet<NuGetFramework>>();
```

Comment:
Added 2 dictionaries( _platformToShortName,  _platformShortToLong ) are initialized values, but never used.
Why is that?

Reply:
oops, it's a leftover from a previous iteration.